man/man8/zfs-receive.8

```diff
@@ -33,9 +32,10 @@
 .Dd February 16, 2020
 .Dt ZFS-RECEIVE 8
 .Os
+.
 .Sh NAME
 .Nm zfs-receive
-.Nd Creates a snapshot whose contents are as specified in the stream provided on standard input.
+.Nd create snapshot with contents specified on standard input
```

Comment:
maybe: "receive snapshots from a backup stream" or similar?

Reply:
Went with "create snapshot from back-up stream", mirroring zfs-send

Reply:
That sounds good, but "backup" no dash.